compute/rn/release-information/release-notes-30-00.adoc

```diff
@@ -225,12 +225,16 @@ By default, the feature is disabled. You can enable the feature by using the PUT
 === Breaking Changes in API
 
 //CWP-42899
-==== Supports pagination for Cloud Discovery API endpoint with backward compatibility
-The *GET, /api/vVERSION/cloud/discovery* API endpoint now displays a paginated response instead of displaying all results on a single page with a backward compatible support for n-2 release versions.
+==== Cloud Discovery API endpoint Updates for Response Pagination
+The *GET, /api/vVERSION/cloud/discovery* API endpoint now displays a paginated response instead of displaying all results on a single page.
 
-Use the query parameters such as *limit* and *offset* to view the paginated response.
+The number of reports retrieved is limited to a maximum of 250, and the default value is 50.
```

Comment:
@ssugandh I defer to you on the accuracy. I do not know the details for this.
 
 In any case it is not 250 reports, correct? Its 250 items per page?
 
 Also if it is for PCCE only we should not add it in this RN for PCCE and not in the PCEE.  If you have concerns, please remove this part, I would like to prevent errors.

Reply:
@Pubs-MV : 
Please see PCCE changes: https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1243/files#diff-8b828544180a1c9e08a2bfea3d85888800000571e788ef351418ffcf50c08b3c

Please see PCEE changes: https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/1243/files#diff-eab25e706f7b928b110b42388e93b24bd10fa4a2697c894fc8730700d7479bb6